cli/cli.go

```diff
@@ -53,5 +53,89 @@ func GetCommands(logger utils.Log) []*clitool.Command {
 				return
 			},
 		},
+		{
+			Name:      "mvn",
+			Usage:     "Collect build-info for a Maven project",
```

Comment:
Collect --> Generate
(Let's implement the same change for the go and gradle commands).